sim/commands.go

```diff
@@ -355,7 +355,7 @@ func queryIsFollowing(srcPuppet, dstPuppet *Puppet) (bool, error) {
 	arg := struct {
 		Source *refs.FeedRef `json:"source"`
 		Dest   *refs.FeedRef `json:"dest"`
```

Comment:
could also remove the `*` here (usually just need pointers for optionals in JSON)

Reply:
good to know! think i'll just keep em as is for now haha